src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -11,38 +11,17 @@
         <cd-table-actions [permission]="permissions.osd"
                           [selection]="selection"
                           class="btn-group"
+                          id="osd-actions"
                           [tableActions]="tableActions">
         </cd-table-actions>
-
-        <div class="btn-group"
-             dropdown>
-          <button type="button"
-                  class="btn btn-sm btn-default btn-label tc_configureCluster"
-                  (click)="configureClusterAction()">
-            <i class="fa fa-fw fa-cog"
-               aria-hidden="true">
-            </i>
-            <ng-container i18n>Set Cluster-wide Flags</ng-container>
-          </button>
-          <button type="button"
-                  dropdownToggle
-                  class="btn btn-sm btn-default dropdown-toggle dropdown-toggle-split">
-            <span class="caret caret-black"></span>
-          </button>
-          <ul *dropdownMenu
-              class="dropdown-menu"
-              role="menu">
-            <li role="menuitem">
-              <a class="dropdown-item"
-                 (click)="configureQosParamsAction()">
-                <i class="fa fa-fw fa-cog"
-                   aria-hidden="true">
-                </i>
-                <ng-container i18n>Set Cluster-wide Recovery Priority</ng-container>
-              </a>
-            </li>
-          </ul>
-        </div>
+        <cd-table-actions [permission]="{read: true}"
+                          [selection]="selection"
+                          onlyDropDown="Cluster-wide configuration"
```

Comment:
Maybe I'm the only one and I also know it wasn't added by this PR but would it make sense to rename it? E.g.:

```suggestion
                          dropDownOnly="Cluster-wide configuration"
```


I'm also fine with a follow-up PR or so.